tests/sanity/test/install/test-installed-files.js

```diff
@@ -31,6 +34,31 @@ describe('verify installed files', function() {
     await sshHelper.executeCommandWithNoError(`test -f ${process.env.ZOWE_ROOT_DIR}/components/jobs-api/bin/jobs-api-server-*.jar`);
   });
 
+  it('fingerprint directory should exist', async function() {
+    await sshHelper.executeCommandWithNoError(`test -d ${process.env.ZOWE_ROOT_DIR}/fingerprint`);
+  });
+
+  it('fingerprint RefRuntimeHash-*.txt should exist', async function() {
+    await sshHelper.executeCommandWithNoError(`test -f ${process.env.ZOWE_ROOT_DIR}/fingerprint/RefRuntimeHash-*.txt`);
+  });
+
+  // # verify the checksums of ROOT_DIR, to self-check zowe-verify-authenticity.sh
+  // $binDir/zowe-verify-authenticity.sh # No parameters! 
+  // You need to 'source' the profile to get JAVA_HOME
+
+  it('fingerprint should match', async function() {
+    let fingerprintStdout = await sshHelper.executeCommandWithNoError(`touch ~/.profile && . ~/.profile && ${process.env.ZOWE_ROOT_DIR}/bin/zowe-verify-authenticity.sh`);
```

Comment:
I'm not sure if ours does, but some linters would complain that this should be a const, not a let

Reply:
Fixed